internal/app/appauth/sharedClients.go

```diff
@@ -0,0 +1,25 @@
+package appauth
```

Comment:
File name suggestion:

`sharedClient.go` -> `client.go`

Camelcase is not commonly used as file names in golang. Golang uses only lowercase (also hyphen is not used) as file names.